components/formats-gpl/src/loci/formats/in/ZeissLSMReader.java

```diff
@@ -343,15 +343,16 @@ public int fileGroupOption(String id) throws FormatException, IOException {
   public short[][] get16BitLookupTable() throws FormatException, IOException {
     FormatTools.assertId(currentId, true, 1);
     if (lut == null || lut[getSeries()] == null ||
-      getPixelType() != FormatTools.UINT16 || validChannels == 0)
+      getPixelType() != FormatTools.UINT16 || channelColor == null)
     {
       return null;
     }
     short[][] s = new short[3][65536];
-    for (int i=2; i>=3-validChannels; i--) {
-      for (int j=0; j<s[i].length; j++) {
-        s[i][j] = (short) j;
-      }
+    Color color = channelColor[prevChannel];
+    for (int j=0; j<s[0].length; j++) {
+      s[0][j] = (short) ((color.getRed() / 255.0) * j);
+      s[1][j] = (short) ((color.getGreen() / 255.0) * j);
+      s[2][j] = (short) ((color.getBlue() / 255.0) * j);
```

Comment:
The idea is to make a gradient, so that relative pixel intensities are preserved when the lookup table is applied.  Not multiplying by ```j``` would set all lookup table entries to be the same, which would result in a blank image.